framework_tvb/tvb/tests/framework/core/adapters/abcadapter_test.py

```diff
@@ -31,163 +31,12 @@
-        We need to make sure that sub-attributes for un-submitted select options are dropped.
-        """
-        kwargs = self.test_adapter.prepare_ui_inputs(self.SUBMIT_DATASET_1)
-
-        for expected_name, expected_type in self.EXPECTED_FILTERED_SET1.items():
-            assert expected_name in kwargs
-            assert isinstance(kwargs[expected_name], expected_type)
-        assert len(self.EXPECTED_FILTERED_SET1) == len(kwargs)
-
-        assert 2 == len(kwargs["monitors_parameters"]["EEG"])
-        assert isinstance(kwargs["monitors_parameters"]["EEG"]["mon_att1"], int)
-        assert isinstance(kwargs["monitors_parameters"]["EEG"]["mon_att2"], float)
-
-    def test_prepare_inputs_wrong_type(self):
-        """
-        Test for ABCAdapter.prepare_ui_inputs, when invalid values passed for numeric fields.
-        """
-        with pytest.raises(Exception):
-            self.test_adapter.prepare_ui_inputs(self.SUBMIT_DATASET_2)
-
-    def test_prepare_inputs_datatype(self, operation_factory):
-        """
-        Test for ABCAdapter.prepare_ui_inputs method when submitting DataType with sub-attributes.
-        """
-        parent_op = operation_factory()
-        test_entity = dao.store_entity(DataType(operation_id=parent_op.id))
-        dataset_3 = {}
-        for key, value in self.SUBMIT_DATASET_3.items():
-            dataset_3[key.replace("$GID$", test_entity.gid)] = value.replace("$GID$", test_entity.gid)
-
-        kwargs = self.test_adapter.prepare_ui_inputs(dataset_3)
-
-        for expected_name, expected_type in self.EXPECTED_FILTERED_SET3.items():
-            assert expected_name in kwargs
-            assert isinstance(kwargs[expected_name], expected_type)
-        assert len(self.EXPECTED_FILTERED_SET3) == len(kwargs)
-
-        assert 2 == len(kwargs["surface_parameters"])
-        assert isinstance(kwargs["surface_parameters"]["att1"], int)
-        assert isinstance(kwargs["surface_parameters"]["att2"], float)
-
-    def test_prepare_select_multiple(self):
-        """
-        Test for ABCAdapter.prepare_ui_inputs method when submitting 2 values in a multiple-select input.
-        """
-        kwargs = self.test_adapter.prepare_ui_inputs(self.SUBMIT_DATASET_4)
-
-        for expected_name, expected_type in self.EXPECTED_FILTERED_SET4.items():
-            assert expected_name in kwargs
-            assert isinstance(kwargs[expected_name], expected_type)
-        assert len(self.EXPECTED_FILTERED_SET4) == len(kwargs)
-
-        assert 2 == len(kwargs["monitors_parameters"]["BOLD"])
-        assert 2 == len(kwargs["monitors_parameters"]["EEG"])
-        assert isinstance(kwargs["monitors_parameters"]["BOLD"]["mon_att1"], int)
-        assert 42 == kwargs["monitors_parameters"]["BOLD"]["mon_att1"]
-        assert 43 == kwargs["monitors_parameters"]["EEG"]["mon_att1"]
-        assert isinstance(kwargs["monitors_parameters"]["BOLD"]["mon_att4"], str)
+        """
```

Comment:
This seems to have empty content now. Will it be replaced or removed entirely?

Reply:
I discussed this with Paula and she said we will come up with some tests in the future and not to delete this class just yet.

Reply:
then, pls leave a TODO with exactly this idea, to be clear for everybody what you two discussed

Reply:
I thought about adding some tests for methods like: build_adapter_from_class, build_adapter, load_with_references, __check_integrity etc. What do you think?

Reply:
Yep. These are all important methods, and should be tested.